docs/language/api.rst

```diff
@@ -1617,6 +1620,15 @@ the quasiquoted form. ``unquote-splice`` is aliased to the ``~@`` symbol.
     `(+ ~@nums)
     ;=> (u'+' 1L 2L 3L 4L)
 
+    `[1 2 ~@(if (< (nth nums 0) 0) nums)]
+    ;=> (u'+' 1L 2L)
```

Comment:
Python 2 is considered legacy support now. We'll likely be removing support for Python 2 soon. #1217. All outputs in the documentation should be in the Python 3 form by then.

I'm not asking you to fix it everywhere in this file, just in the section you've added to to make it consistent. I'll even approve the pull without this particular fix, but we might as well do it now instead of making it worse.